autopush/websocket.py

```diff
@@ -1000,7 +1000,7 @@ def process_register(self, data):
         self.transport.pauseProducing()
 
         d = self.deferToThread(self.ap_settings.make_endpoint, self.ps.uaid,
```

Comment:
Do you have an integration test for this, by chance? I don't want to block this from landing based on that, but it would be nice to see it end-to-end.